.gitignore

```diff
@@ -1,7 +1,9 @@
-__pycache__/
-*.pyc
-/.cache_metadata.json
-.coverage
-/.venv/
-/data/
-/cache/
+__pycache__/
+*.pyc
+/.cache_metadata.json
+.coverage
+/.venv/
+/data/
+/cache/
+/.vscode/

```

Comment:
Please add IDEs to your global git config; otherwise this list will become really lengthy really quick :P